dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Sleep/SleepTaskDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import { NodeRun, SleepNode } from 'littlehorse-client/proto'
+import { FC } from 'react'
+import { NodeDetails } from '../NodeDetails'
+import { convertSecondsToReadableTime } from '../../../../../utils'
+
+export const SleepTaskDetails: FC<{ sleepNode?: SleepNode; nodeRun?: NodeRun }> = ({ sleepNode, nodeRun }) => {
```

Comment:
It should be called just SleepDatails

Reply:
it is fixed

Reply:
I believe he was referring to the function name `SleepTaskDetails:` -> `SleepDetails:`